setup-env-file.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+function generate_key() {
+    dd if=/dev/urandom bs=32 count=1 2>/dev/null | base64 | tr -d -- '\n' | tr -- '+/' '-_' ; echo
+}
+
+export CLIENT_SECRET=$(generate_key)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)